src/Symfony/Component/Translation/MessageCatalogue.php

```diff
@@ -65,10 +65,14 @@ public function getDomains()
     /**
      * {@inheritdoc}
      */
-    public function all(string $domain = null)
+    public function all(string $domain = null, $strict = false)
```

Comment:
adding an argument would break BC, we need to find another idea

Reply:
Right, I forget BC, sorry.

Better idea, I can come with at the moment, is to return intl-icu domain if requested and then do diff for all and intl-icu messages.

Please review new version